### PR TITLE
Add Buyer related classes and test cases

### PR DESCRIPTION
**Pull request by zengzihui**
Edit buyer related classes such as the command string
Add test cases for buyer related classes

### CODE REVIEW

**Review by KhoonSun47 — APPROVED** [review]
LGTM! Good luck!

File: src/main/java/seedu/address/logic/parser/AddSellerCommandParser.java [hunk 1]
```diff
@@ -47,25 +47,25 @@ public AddSellerCommand parse(String args) throws ParseException {
...
+        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_PHONE, PREFIX_HOUSING_TYPE, PREFIX_LEVEL,
                 PREFIX_EMAIL, PREFIX_BLOCK, PREFIX_STREET, PREFIX_UNITNUMBER, PREFIX_POSTALCODE);
         Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
         Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         UnitNumber unitNumber = ParserUtil.parseUnitNumber(argMultimap.getValue(PREFIX_UNITNUMBER).get());
         PostalCode postalCode = ParserUtil.parsePostalCode(argMultimap.getValue(PREFIX_POSTALCODE).get());
         Street street = ParserUtil.parseStreet(argMultimap.getValue(PREFIX_STREET).get());
-        String housingtype = ParserUtil.parseHousing(argMultimap.getValue(PREFIX_HOUSINGTYPE).get());
+        String housingtype = ParserUtil.parseHousing(argMultimap.getValue(PREFIX_HOUSING_TYPE).get());
```
Comment (lokidoki102) [inline src/main/java/seedu/address/logic/parser/AddSellerCommandParser.java:68]: `housingType`?

Comment (zengzihui) [reply src/main/java/seedu/address/logic/parser/AddSellerCommandParser.java:68]: ohh will change it in the next PR

**Review by felixchanyy — APPROVED** [review]
LGTM

**Review by lokidoki102 — APPROVED** [review]
looks good to me 👍

**Review by KhoonSun47 — APPROVED** [review]
LGTM

**Pull request merged**